### PR TITLE
VIH-7883 - Validation on edit Judge name

### PR DESCRIPTION
**Pull request by RosDiaf**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7883

### Change description ###
Validation on edit Judge name to allow only alphanumeric characters

edit-judge-name

### CODE REVIEW

**Review by twyfordr273 — DISMISSED** [review]
Approved

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts [hunk 1]
```diff
@@ -123,6 +124,7 @@ export class JudgeParticipantStatusListComponent extends WRParticipantStatusList
 
     async saveJudgeDisplayName() {
         this.judge.display_name = this.newJudgeDisplayName;
+        this.judge.display_name = new SafePipe().transform(this.judge.display_name);
```
Comment (phillipjchambers) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts:127]: Pipes should be used in templates. If it doesn't need to be used in a template, like this
```
{ someValue | safe }
```
then it doesn't need to be a pipe & should just be a func.

Comment (RosDiaf) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts]: Pipe removed

**Review by phillipjchambers — APPROVED** [review]

**Review by twyfordr273 — APPROVED** [review]
Approved

**Review by phildebrah — APPROVED** [review]
Tested and working successfully

**Pull request merged**